instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcInstrumenterFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jdbc.internal;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.db.DbClientSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.db.SqlClientAttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetClientAttributesExtractor;
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public final class JdbcInstrumenterFactory {
+  public static final String INSTRUMENTATION_NAME = "io.opentelemetry.jdbc";
+  private static final JdbcAttributesGetter dbAttributesGetter = new JdbcAttributesGetter();
+  private static final JdbcNetAttributesGetter netAttributesGetter = new JdbcNetAttributesGetter();
+
+  public static Instrumenter<DbRequest, Void> createInstrumenter() {
+    return createInstrumenter(null);
+  }
+
```

Comment:
Is this used? `null` is not a valid value for `OpenTelemetry` anyway, it's gonna throw an NPE if somebody calls it
```suggestion
```

Reply:
Yeah, my bad... forgot to add a `GlobalOpenTelemetry.get()` call.